app/views/content/landing/social-mailing-list-sign-up/_mailing-list.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <div class="grey col-space-l">
+    <section class="col col-720 col-space-0">
+      <%= render Content::MailingListComponent.new(
+        title: "Get free advice and support that's tailored to you",
+        color: "transparent",
+        margin: false,
+        heading: :l
+      ) do %>
+
+        <p>
+          Whether you’re just thinking about teaching or you’re ready to apply, we’re here to help guide you through the steps to becoming a teacher. Discover how exciting and rewarding teaching can be.
+        </p>
+
+        <p>
+          Sign up for emails and get:
```

Comment:
I think this is a bit misleading - you don't "get" all these things by signing up to the mailing list, you just get the first one and then more info about how to do the others

The wording on the existing page is `Find out how to get:` which is a bit more accurate (although even this could probably be improved)